oc2ls.md

```diff
@@ -749,7 +753,16 @@ The behavior of an implementation receiving an OpenC2 Response with an unsupport
 #### 3.4.1.6 Features
 | Type Name | Base Type | Description |
 | :--- | :--- | :--- |
-| **Features** | ArrayOf(Feature) | An array of zero to ten names used to query an actuator for its supported capabilities. |
+| **Features** | ArrayOf(Feature) [0..10] | An array of zero to ten names used to query an actuator for its supported capabilities. |
+
+**Usage Requirements:**
+
+* A Producer MUST NOT send a list containing more than one instance of any Feature.
```

Comment:
I think this should be "A Consumer MUST NOT ..."

Reply:
The producer will send an OpenC2-Command containing a features target that is the list of features of interest (e.g., "features": ["profiles", "pairs"]).

The consumer will send an OpenC2-Response containing the information requested by the producer, e.g., the versions, profiles, pairs, and/or schema properties of the response.